src/metatracer.rs

```diff
@@ -42,14 +42,17 @@ const PHASE_COUNTING: u32 = 0b00; // The value specifies the current hot count.
 const PHASE_TRACING : u32 = 0b01;
 const PHASE_COMPILED: u32 = 0b10;
 
-/// A LocPhase stores data that the `MetaTracer` needs internally to determine if and when to trace
-/// a given location, and whether a compiled trace already exists for that location.
-pub struct LocPhase {
+/// A `Location` identifies a control point position in the end-user's program (the `MetaTracer`
```

Comment:
Shall we mention that the identity of a `Location` matters? That passing two equivalent `Locations` will not do what the user expects?

Obvious to us, I know.

Reply:
How about "A `Location`uniquely identifies"?

Reply:
Is that strictly true? I'd say that the address of a `Location` uniquely identifies it, but even that is horrible to offer up to a user as an explanation.

I guess I'm trying to guard against people thinking that they could materialise `Locations` on the fly without storing them. I could totally see myself try something silly like that.

Reply:
Yes, it's true. [The fact that we only let people give us references to the location is kind-of secondary.] But you're right, it could be misread. Let me try again.

Reply:
[Indeed, we don't care if you change the address of a Location, provided you give us the same contents.]

Reply:
good point.